services/meta/client.go

```diff
@@ -983,6 +983,12 @@ func snapshot(path string, data *Data) error {
 		return err
 	}
 
+	if _, err := os.Stat(file); err == nil {
```

Comment:
I can move it to client_windows.go, but then every time we make a change to client package, we need to edit it twice. Better option it to make snapshot to a separate package, and create two files. Will that be okay?